package.xml

```diff
@@ -1,34 +1,42 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>mas_navigation_tools</name>
   <version>0.0.2</version>
   <description>several base navigation tools</description>
 
-  <maintainer email="frederik.hegger@h-brs.de">Frederik Hegger</maintainer>
-  <maintainer email="olima_84@yahoo.com">Oscar Lima</maintainer>
+  <maintainer email="robotics@mail.inf.h-brs.de">MAS Robotics</maintainer>
 
   <license>GPLv3</license>
 
   <author email="frederik.hegger@h-brs.de">Frederik Hegger</author>
   <author email="olima_84@yahoo.com">Oscar Lima</author> <!--path length calculator-->
+  <author email="suvich15@todo.todo">Sushant Chavan</author>
```

Comment:
Could you please change the email to `sushant.chavan@smail.inf.h-brs.de`. I think we forgot about this when we released the tool.